bin/migrations-run

```diff
@@ -16,6 +16,18 @@ def main():
     migration_names = {path.stem for path in migration_files}
 
     with get_connection() as conn:
+        # Bootstrap first migration
+        conn.execute(sql.text("""
+            CREATE TABLE IF NOT EXISTS MigrationVersions (
+                id         BIGINT AUTO_INCREMENT,
+                name       VARCHAR(255),
+                migratedAt DATETIME DEFAULT CURRENT_TIMESTAMP,
+
+                PRIMARY KEY (id)
+            );
+        """))
+        conn.commit()
```

Comment:
The migration table should be bootstrapped in the "run" script, otherwise it couldn't list all migrations that were/weren't run.